devel/libenkf/src/hook_manager.c

```diff
@@ -131,14 +130,14 @@ void hook_manager_add_config_items( config_parser_type * config ) {
 
     item = config_add_schema_item( config , QC_WORKFLOW_KEY , false );
     config_schema_item_set_argc_minmax(item , 1 , 1 );
-    config_schema_item_iset_type( item , 0 , CONFIG_EXISTING_PATH );
```

Comment:
I thought you wanted a change in behavior for QC_WORKFLOW as well. Will keep old variant with path for compatibility.